lib/rails/projections.json

```
@@ -0,0 +1,14 @@
+{
+  "spec/factories/*.rb": {
+    "affinity": "model",
+    "alternate": "app/models/{}.rb",
+    "command": "factory",
+    "rubyMacro": ["factory", "sequence", "trait", "transient"],
+    "template": "FactoryBot.define do\n  factory :{} do\n  end\nend",
+    "test": [
+      "spec/models/{}_test.rb",
```

I figured you meant this instead?

```suggestion
      "spec/models/{}_spec.rb",
```

while we’re at it, I have a vague recollection this is deprecated by projectionist and possibly bundler.vim?